requirements/requirements.txt

```diff
@@ -19,8 +19,5 @@ pythonnet==2.5.2
 pyteomics==4.4.2
 fastcore==1.3.21
```

Comment:
A few comments about requirements in general:
* All of these are very strict and the setup allows to do it loose as well. Notice that there is nice midway to work with e.g. `numpy>=1.19` or perhaps even more stable `numpy~=1.19`. This should still be perfectly compatible with sta stable and loose option, but just make the stable a little more flexible. Taking this one step further, we can also go for e.g. `stable` and `minimal` options. Where `stable` enforces the `==` from the requirements.txt, minimal replaces this with `>=` and no extra just allows any.
* This list is fairl long, presumably, some of them are not essential. For instance: `fastcore` and `wget` are probably develop requirements, not main.
* While I understand it is good to control versions like this, I have found that it is easy to introduce conflicts. An example might be `numba` or `pandas` with `numpy`. Note that both `numba` and `pandas` actually have `numpy` as a dependancy. In principle, numpy can thus be left off the list, since it will always be installed anyways and we can avoid potential conflicts. That said, for the `stable` options it can be useful, but than we can make the argument that we should actually do a `pip freeze` and version pin all other subdependancies just as well...